### DISCUSSION TITLE
How can I Create table without header columns ?

### DISCUSSION THREAD

**Discussion by Arman-Espiar** [spectreconsole/spectre.console#816] [Q&A] [answered]
Hi everyone, I want to show a matrix 3*3 (sodoku) in the table but a table needs the header for each column, But I need rows. How can I create a sodoku table?

**Comment by patriksvensson** [comment, marked as answer]
@Arman-Espiar You can use the [HideHeaders](https://spectreconsole.net/api/spectre.console/tableextensions/b6f4b37c) extension method for `Table` to hide the headers. You will still need to add columns to the table though.
